entity-management-web/src/main/java/eu/europeana/entitymanagement/web/service/ZohoSyncService.java

```diff
@@ -366,15 +398,29 @@ private void performCreateOperations(
     }
 
     List<String> entitiesToUpdate = performEntityRegistration(createOperations, zohoSyncReport);
+    
```

Comment:
performEntityRegistration for the individual entities needs to be updated, and the code that beuild the EntityId based on ZohoId needs to be removed. Instead the identifier needs to be generated as for the other entities using the sequence. The update of ZohoRecord with the EuropeanaID needs to be invoked during the creation, before saving the EntityRecord into the database

Reply:
see requirements:
Creation workflow:

    generate a new Europeana identifier using the same approach as for other entities (sequential)

    Set the value of the new “Europeana ID” field in Zoho with the newly generated identifier for the entity

    Keep the Zoho URL as ID of the Zoho Proxy (as it is currently done)

    Update the entity data